### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by hybla0** [Toiongo/VK_Audio_Download_Ext#2]
Hi,
I tried the extension but it downloads a Russian audio MP3 of few seconds, that seems to be a kind of message, instead of the actual track.
@Toiongo Do you know of any solution?

**Comment by Toiongo** [comment]
Are you sure you're on the music page, logged in and trying to download a track that's not behind a paywall ? I've just tried to get "Joji - Gimme Love" this way and it went fine.

**Comment by hybla0** [comment]
Yes I was on the music page, where you can search tracks.
What do you mean by "track behind a paywall"?
Are there differences between tracks?

**Comment by Toiongo** [comment]
Yes, some tracks won't be available for download even with the extension, as they're protected by a paywall and aren't available for free of charge listening.
